src/inference/dev_api/openvino/runtime/make_tensor.hpp

```diff
@@ -51,6 +52,10 @@ OPENVINO_RUNTIME_API std::shared_ptr<ITensor> make_tensor(const std::shared_ptr<
                                                           const Coordinate& begin,
                                                           const Coordinate& end);
 
+OPENVINO_RUNTIME_API std::shared_ptr<ITensor> make_tensor(const std::shared_ptr<IRemoteTensor>& other,
+                                                          const Coordinate& begin,
+                                                          const Coordinate& end);
```

Comment:
do we need it? I supposed that overload with ITensor can also work for  IRemoteTensor

Reply:
Seems so, I removed extra function and reused an existing one for both remote and ordinary tensors